deploy/helm/csi-wekafsplugin/Chart.yaml

```diff
@@ -5,14 +5,13 @@ maintainers:
   - name: WekaIO, Inc.
     email: csi@weka.io
     url: https://weka.io
-sources:
```

Comment:
Why removing from here?

Reply:
the field is optional which should lead to the project, not to a release. And the Homepage link is already pointing to the project/repo (with all releases) @sergeyberezansky 

![image](https://user-images.githubusercontent.com/119401705/224715367-246b3b9f-7637-4ca1-bed3-bde81e41e4df.png)

Reply:
updated it back to point to a tag